index.html

```diff
@@ -2156,6 +2156,29 @@
   </div>
   <!-- ends Generate Video Fingerprint -->
 
+  <!-- Strip metadata -->
+  <span data-toggle="modal" data-target="#strip_metadata"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Strip metadata">Strip metadata</button></span>
+  <div id="strip_metadata" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Strips metadata from video file</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -vcodec copy -acodec copy test-nometadata.mov</code></p>
```

Comment:
Either we add a note explaining that `-vcodec` and `-acodec` is the same than `-c:v` and `-c:a`, or we unify.